package.json

```diff
@@ -2411,15 +2411,18 @@
             },
             {
                 "command": "aws.amazonq.transformationHub.summary.reveal",
-                "title": "%AWS.command.q.transform.showChangeSummary%"
+                "title": "%AWS.command.q.transform.showChangeSummary%",
+                "enablement": "gumby.reviewState == InReview"
             },
```

Comment:
Doing this via "enablement" makes it consistent with the "Show Plan" button in the dropdown, which is visible the entire time but enabled/disabled as appropriate.

Reply:
yes, that is a strongly preferred UI practice.